roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -124,14 +129,18 @@ def local_images(self, images, task_vars):
         ]
 
     def is_image_local(self, image, task_vars):
-        result = self.module_executor("docker_image_facts", {"name": image}, task_vars)
+        """Check if image is already in local docker index."""
+        result = self.execute_module("docker_image_facts",
+                                     module_args={"name": image},
+                                     task_vars=task_vars)
```

Comment:
Yep. I don't know why that ever worked :)

If you don't want to clutter things up with named args that Ansible will never change, I can change that back.